editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -3015,6 +3015,86 @@ export var storyboard = (props) => {
               `),
           )
         })
+
+        it('applies flex props if the conditional is in a flex container', async () => {
+          const renderResult = await renderTestEditorWithCode(
+            makeTestProjectCodeWithSnippet(`<div
+            style={{
+              position: 'relative',
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 114.5,
+              width: 'max-content',
+              height: 'max-content',
+            }}
+            data-uid='root'
+          >
+            {
+              // @utopia/uid=conditional
+              true ? null : null
+            }
+            <img
+              style={{
+                width: 100,
+                height: 100,
+                position: 'absolute',
+                top: 120,
+                left: 0,
+              }}
+              data-uid='bbb'
+              src='/editor/utopia-logo-white-fill.png?hash=d7275eef10f8344f4b52a4f5ba1c92e698186d61'
+            />
+          </div>`),
+            'await-first-dom-report',
+          )
+          await selectComponentsForTest(renderResult, [makeTargetPath('root/bbb')])
+          await pressKey('x', { modifiers: cmdModifier })
+
+          await selectComponentsForTest(renderResult, [makeTargetPath('root/conditional/a25')])
+
+          const canvasRoot = renderResult.renderedDOM.getByTestId('canvas-root')
+
+          firePasteEvent(canvasRoot)
+
+          // Wait for the next frame
+          await clipboardMock.pasteDone
+          await renderResult.getDispatchFollowUpActionsFinished()
+
+          await pressKey('Esc')
+          await renderResult.getDispatchFollowUpActionsFinished()
+
+          // no position: absolute, top, left in the pasted image
```

Comment:
you could expect() this from computedStyle or allElementProps!

Reply:
great idea, https://github.com/concrete-utopia/utopia/pull/4374/commits/585aa88bb955ae5813611e7ac79607ef9cfe5514